docs/Design.md

```diff
@@ -5,19 +5,19 @@ This document describes the main components within the VLCB library and how they
 This VLCB library is based on Duncan Greenwood's [CBUS library](https://github.com/MERG-DEV/CBUS)
 and extended with VLCB specific features.
 
-This library is still in progress.
+This library is still work in progress.
 
 ## High Level Architecture
-The code is organized as a central controller object that controls functionality
+The code is organised as a central controller object that controls functionality
 via a storage object and a set of service objects.
 
-Services implement various groups of functionalities within VLCB such as events or DCC control.
-The user sketch can select the set of services needed to provide the functionality
-that is necessary for the VLCB module that is created.
+Services implement various groups of functionality within VLCB such as events or DCC control.
+The user sketch can select the set of services needed to provide the necessary functionality
+for the VLCB module that is created.
 There are also services for user interfaces and communication over different transports such as CAN, Wifi and BLE.
 Currently only a CAN transport using the CAN2515 chip is included in the library.
 
-The library supports a set of storage for node variables and event variables in EEPROM or Flash memory.
+The library supports storage for node variables and event variables in EEPROM or Flash memory.
```

Comment:
Good, but I miss the bit about that different types of EEPROM storages are supported by the library..

Reply:
I think that this gets covered lower down in "Configuration".